src/StaticFAQ/FAQArticle.js

```diff
@@ -38,7 +38,7 @@ const FAQArticle = (props: Props) => (
   >
     <Card>
       <div>
-        <Heading weight="medium" size="small">
+        <Heading weight="medium" size="small" data-cy="faq-title">
```

Comment:
Where is this `faq-title` used in the tests?

Reply:
Ups I dont use it - https://github.com/kiwicom/smart-faq/pull/424/files#diff-990a76d361172edce79bfd1e383eb265R27